src/sidebar/services/groups.js

```diff
@@ -20,6 +20,9 @@ var serviceConfig = require('../service-config');
 // @ngInject
 function groups($rootScope, store, api, isSidebar, localStorage, serviceUrl, session,
                 settings) {
+
+  // The document URI passed to the most recent `GET /api/groups` call in order
+  // to include groups associated with this page.
```

Comment:
I wonder if it might be helpful to "future devs" to include in this comment _why_ this value is retained? I'm not sure I'd pick it up easily in isolation (without knowing about the loading logic).

I note in the expanded full module this declaration is made quite a distance from the assignment of a value; this is likely necessary for scope/hoisting reasons, but there you go :)

Reply:
Agreed on both points. I'll move the declaration down and expand the comment.